pkg/proxy/nftables/proxier.go

```diff
@@ -336,6 +340,7 @@ var nftablesBaseChains = []nftablesBaseChain{
 	{"filter-input", knftables.FilterType, knftables.InputHook, knftables.DNATPriority + "-10"},
 	{"filter-forward", knftables.FilterType, knftables.ForwardHook, knftables.DNATPriority + "-10"},
 	{"filter-output", knftables.FilterType, knftables.OutputHook, knftables.DNATPriority + "-10"},
+	{"filter-output-post-dnat", knftables.FilterType, knftables.OutputHook, knftables.FilterPriority},
```

Comment:
(Hm... I don't like the fact that it's not really obvious that "FilterPriority" is after "DNATPriority"... maybe I just switch to all numeric priorities...)

Reply:
I think it is fine, anyone working on NFT should be aware of conntrack, dnat, filter, snat priorities ...

(random thought: maybe we should do filtering even before conntrack, to apply it on runtime  (already established connections) 😅 )